api/src/main/java/org/pmiops/workbench/tools/cdrconfig/AccessTierConfig.java

```diff
@@ -0,0 +1,19 @@
+package org.pmiops.workbench.tools.cdrconfig;
+
+// used by UpdateCdrConfig
+// TODO: enable mapstruct in api/tools and move this class there
+
+// used for JSON parsing of cdr_config_*.json
+// adapted from DbAccessTier
+public record AccessTierConfig(
```

Comment:
convert these from "Value Objects" to records, which enforce that all fields are considered on creation - though they can be null